File: testdata/java/overloaded_methods_with_zero_args.java
class Runner {
    public void run() {
        System.out.println("Running once");
    }
    
    public void run(int times) {
        for (int i = 0; i < times; i++) {
            System.out.println("Running iteration " + i);
        }
    }
    
    public void test() {
        this.run();
        this.run(5);
    }
}

File: testdata/go/overloaded_methods_with_zero_args.go
package converted

type runner struct {
}

func newRunner() runner {
	this := runner{}
	return this
}

func (this *runner) Run() {
	// migrated from overloaded_methods_with_zero_args.java:2:5
	System.out.println("Running once")
}

func (this *runner) RunWithInt(times int) {
	// migrated from overloaded_methods_with_zero_args.java:6:5
	i := 0
	for ; i < times; i++ {
		System.out.println(("Running iteration " + i))
	}
}

func (this *runner) Test() {
	// migrated from overloaded_methods_with_zero_args.java:12:5
	this.Run()
	this.RunWithInt(5)
}
